rust/alloc/raw_vec.rs

```diff
@@ -465,7 +515,7 @@ impl<T, A: Allocator> RawVec<T, A> {
         Ok(())
     }
 
-    #[cfg(not(no_global_oom_handling))]
+    // TODO: Why this was marked as `#[cfg(not(no_global_oom_handling))]`?
```

Comment:
Good question. Doesn't seems like it needs it to me either.

Reply:
Thanks for taking a look -- I was not sure if I was missing something. I can drop the comment then, and submit the change upstream.

Reply:
Looks unnecessary to me as well.

Reply:
Thanks both! Opened https://github.com/rust-lang/rust/pull/86755